### DISCUSSION TITLE
Issue when scanning images which are not up to date with the image with the same tag on dockerhub

### DISCUSSION THREAD

**Discussion by nilshermansson-op** [aquasecurity/trivy#5307] [Q&A] [answered]
### Question

I'm currently working on a pipeline on azure which should run every day and collect vulnerabilities detected in images which are hosted on ACR. All images are updated weekly using another pipeline.

A problem has arisen which causes the scans to fail occasionally. This can be fixed by pulling the image on Dockerhub to ACR and running the scan again. It produces an error which says: `MANIFEST_UNKNOWN: manifest`

I suspect this is because a patch has been released which means the image on dockerhub under the same tag as the one I have on ACR don't match with the one I have on ACR anymore.

My question is: is there a way to scan an image despite it not being up to date with the image which is on dockerhub? I realise one solution to this problem is to simply pull all images before scanning but I'd like to avoid this if possible.

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

Azure

### Version

```bash
Version: 0.45.1
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-10-02 12:19:51.756133046 +0000 UTC
  NextUpdate: 2023-10-02 18:19:51.756132446 +0000 UTC
  DownloadedAt: 2023-10-02 13:14:46.469938572 +0000 UTC
```

**Comment by knqyf263** [comment, marked as answer]
`trivy image your_image@sha256:xxx` should help

**Reply by nilshermansson-op** [reply]
Thanks!
